static/templates/status_emoji.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if status_emoji_info}}
+    {{#if status_emoji_info.emoji_alt_code}}
+        <div class="emoji_alt_code">&nbsp:{{status_emoji_info.emoji_name}}:</div>
+    {{else}}
+        {{#if status_emoji_info.still_url}}
+        <img src="{{status_emoji_info.still_url}}" class="emoji status_emoji" data-animated-url="{{status_emoji_info.url}}" data-still-url="{{status_emoji_info.still_url}}" />
+        {{else if status_emoji_info.url}}
+        <img src="{{status_emoji_info.url}}" class="emoji status_emoji" data-animated-url="{{status_emoji_info.url}}" data-still-url="{{status_emoji_info.still_url}}" />
+        {{else}}
+        <div class="emoji status_emoji emoji-{{status_emoji_info.emoji_code}}"></div>
+        {{/if}}
+    {{/if}}
+{{/if}}
```

Comment:
I would usually expect this sort of template extraction to be passed `status_emoji_info` from the caller / calling template, and just access fields like `emoji_code` here.

Reply:
Done.

Also noticed that we were inserting `data-still-url="{{still_url}}"` in the case where `{{#if still_url}}` was false so stopped doing that.